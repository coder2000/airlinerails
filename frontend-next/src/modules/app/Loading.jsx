// @flow

import * as React from 'react';
import { Box } from 'bloomer';

export default function Loading() {
  return <Box>Loading...</Box>;
}
