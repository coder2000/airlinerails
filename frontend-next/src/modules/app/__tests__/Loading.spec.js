import * as React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';

test('Loading component renders without error', () => {
  const loading = shallow(<Loading />);

  expect(loading).toMatchSnapshot();
});
