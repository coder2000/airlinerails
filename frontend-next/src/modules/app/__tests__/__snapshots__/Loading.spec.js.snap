// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders without error 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": "Loading...",
      "hasTextAlign": "centered",
    },
    "ref": null,
    "rendered": "Loading...",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "Loading...",
        "hasTextAlign": "centered",
      },
      "ref": null,
      "rendered": "Loading...",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
